[Bug]: build failure after deps update

### Steps to reproduce

Have dependabot bump dependencies and try to build the project

### Expected results

Error: src/builders/puppeteer/index.ts(205,1): error TS2742: The inferred type of 'default' cannot be named without a reference to '@angular-devkit/architect/node_modules/@angular-devkit/core'. This is likely not portable. A type annotation is necessary.


### Actual results

No errors

### @puppeteer/ng-schematics version

xxx

### Angular CLI version

xxx

--- a/.github/dependabot.yml
+++ b/.github/dependabot.yml
@@ -10,6 +10,7 @@ updates:
     ignore:
       - dependency-name: devtools-protocol
       - dependency-name: '@types/node'
+      - dependency-name: '@angular-devkit/*'
     groups:
       dependencies:
         dependency-type: production

